### PR TITLE
fix 'View Replays in Vault' change to 'Online Vault' Tab in Replays

### PR DESCRIPTION
**Pull request by GrotheFAF**
fixes (2nd point in) #414

- [x] PR branch should be named `issuenum`-`fix`/`feature`/`cleanup`-`description`
- [x] Code is split into logical commits
- [x] Code has tests
- [x] Final commit includes "Fixes #issue" in commit message

When all builds pass and a maintainer is happy with your PR, the "ready" label will be applied. Please complete these tasks then:

- [x] Rebase onto develop
- [x] Add changelog entry
- [ ] Remove this entire template section

### CODE REVIEW

File: src/chat/chatter.py [hunk 1]
```diff
@@ -387,6 +387,7 @@ def viewReplay(self):
     @QtCore.pyqtSlot()
     def viewVaultReplay(self):
         """ see the player replays in the vault """
+        self.lobby.client.replays.setCurrentIndex(2)  # focus on Online Fault
```
Comment (duk3luk3) [inline src/chat/chatter.py:387]: Is there a reason this isn't grouped with the `self.lobby.client.mainTabs.setCurrentIndex` call?

**Comment by duk3luk3** [comment]
Tested, does what it says.

Before this change: Selecting "View Replays in Vault" in a chat member's context menu switched to whatever Replay Vault Tab was selected last. 
After: It always switches to the "Online Vault" Tab so you can actually see what you wanted to see.

In the future, please put a description like this into PRs.

File: src/chat/chatter.py (hunk 1 above)
Comment (GrotheFAF) [reply src/chat/chatter.py:387]: the logic behind being first is: 1.get there, 2. do stuff.

Comment (duk3luk3) [reply src/chat/chatter.py:387]: but then `self.lobby.client.mainTabs.setCurrentIndex` should also be part of "get there", shouldn't it?

Comment (GrotheFAF) [reply src/chat/chatter.py:387]: well, if I would have moved the line, then the question would have been why I move the line.
I can't win here.

Comment (duk3luk3) [reply src/chat/chatter.py:387]: Where do you think the line should be?

Comment (GrotheFAF) [reply src/chat/chatter.py:387]: Question of taste, does it matter?

Comment (duk3luk3) [reply src/chat/chatter.py:387]: you said yourself that it should be "1. get there, 2. do stuff".

Comment (GrotheFAF) [reply src/chat/chatter.py:387]: so?

Comment (GrotheFAF) [reply src/chat/chatter.py:387]: again, does it matter?

Comment (Wesmania) [reply src/chat/chatter.py:387]: `self.lobby.client.mainTabs.setCurrentIndex` switches client tab to the replays tab, it's conceptually different than setting up the replay tab. It's fine the way it is, go ahead and merge it.

**Comment by duk3luk3** [comment]
Please rebase and add changelog entry.

**Pull request merged**